src/app/pages/dashboard/job-result/job-result.component.html

```diff
@@ -18,49 +18,7 @@ <h5 class="d-flex justify-content-between">
                 <nb-icon icon="globe-outline"></nb-icon>
                 {{ jobTableData?.observable_name }}
               </span>
-              <span>
-                <!-- Job Status Icon -->
-                <intelowl-job-status-icon
-                  [value]="jobTableData?.status"
-                  rowData=""
-                ></intelowl-job-status-icon>
-                <!-- Divider -->
-                <p class="vr mx-2"></p>
-                <!-- Download Sample btn -->
-                <nb-icon
-                  *ngIf="jobTableData.is_sample"
-                  class="dl-btn"
-                  nbTooltip="Download File"
-                  icon="download-outline"
-                  (click)="getJobSample()"
-                ></nb-icon>
-                <!-- Raw JSON -->
-                <nb-icon
-                  class="dl-btn"
-                  nbTooltip="View raw JSON"
-                  icon="code-outline"
-                  (click)="getJobRawJson()"
-                >
-                </nb-icon>
-                <!-- Kill Job Button -->
-                <nb-icon
-                  *ngIf="jobTableData.status === 'running'"
-                  class="dl-btn"
-                  status="warning"
-                  nbTooltip="kill running analyzers and mark as 'killed'"
-                  icon="slash"
-                  (click)="killJob()"
-                >
-                </nb-icon>
-                <!-- Delete Job Button -->
-                <nb-icon
-                  status="danger"
-                  nbTooltip="Delete Job"
-                  icon="trash-2-outline"
-                  (click)="deleteJob()"
-                >
-                </nb-icon>
-              </span>
+              <intelowl-job-actions-menu> </intelowl-job-actions-menu>
```

Comment:
need to pass the `job` object from JobResult to new component. Use @Input decorator on the new component for receiving this.

Reply:
> I'm bit unsure now as to how the program flow will be and how I should connect the 2 components so that I can use this new one inside the old JobResults one.

@m0mosenpai. You've used the HTML tag correctly. Just need to pass the job object from JobResult component to the actions component (using the angular's @Input decorator to receive the prop). Angular has bit of a learning curve, so make sure to read about the basics. Like, how to structure the DOM, how to pass data from components, @Input, @Output, @Event decorators, etc.